### PR TITLE
Installer configure php, to enable opcache and httpOnly

### PR DESCRIPTION
**Pull request by D9ping**
Let the installer reconfigure the php configuration.
Turn on httpOnly session cookies flags.
~~Turn on session.use_strict_mode.~~
Turn on php7 zend opcache if on php7.

### CODE REVIEW

**Comment by D9ping** [comment]
This features to make the default php configuration by the RaspAP installer more secure: enable httpOnly on session id cookies, so it cannot easilty be haijcked by javascript.
and RaspAP faster(opcache) is all optional.
I removed the code for turning off of php extensions because it's unknown if they may be needed in the future.

**Comment by billz** [comment]
Very nice additions @D9ping. Some observations:

Until now we haven't modified php-cgi settings with the installer. The configuration changes made here might not adversely impact other installed packages, but experience has shown that users can and do run RaspAP alongside all kinds of things on the RPi. For this reason, it's a good idea to inform the user what's happening during the install and prompt them to continue with (or optionally, skip) these changes.   

Great to see that the php configuration is backed up in `reconfigure_php()`. Better still if we standardize by backing up to `$raspap_dir/backups/` as with the other configs.

Ideally, the above backup would be also uninstalled by `check_for_backups()`  in `uninstall.sh`.

File: installers/uninstall.sh [hunk 1]
```diff
@@ -68,14 +82,21 @@ function check_for_backups() {
...
+        fi
         if [ -f "$raspap_dir/backups/rc.local" ]; then
             echo -n "Restore the last rc.local file? [y/N]: "
             read answer
             if [[ $answer -eq 'y' ]]; then
                 sudo cp "$raspap_dir/backups/rc.local" /etc/rc.local
             else
                 echo -n "Remove RaspAP Lines from /etc/rc.local? [Y/n]: "
-                if $answer -ne 'n' ]]; then
+                if [[ $answer -ne 'n' ]]; then
```
Comment (D9ping) [inline installers/uninstall.sh]: Missing [ here, so remove RaspAP Lines from /etc/rc.local did not work.

**Comment by billz** [comment]
Resolves #225. Nice work, thanks.

**Pull request merged**